src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/email/ReportSender.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.services.email;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.reports.ReportsService;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.reports.ZipFileSummaryResponse;
+import uk.gov.hmcts.reform.bulkscanprocessor.util.CsvWriter;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.List;
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+public class ReportSender {
+
+    private static final Logger log = LoggerFactory.getLogger(ReportSender.class);
+
+    public static final String EMAIL_SUBJECT = "Bulk Scan daily report";
+    public static final String EMAIL_BODY = "This is an auto generated email. Do not respond to it.";
+    public static final String ATTACHMENT_PREFIX = "bulk_scan_envelopes_";
```

Comment:
suggestions for alternative email title/body/attachment welcome

Reply:
`zip-file-summary-`?